pkg/simple/client/mysql/mysql.go

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2018 The KubeSphere Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mysql
+
+import (
+	"fmt"
+	"github.com/gocraft/dbr"
+	"k8s.io/klog"
+)
+
+type MySQLClient struct {
+	database *Database
+}
+
+func NewMySQLClient(option *MySQLOptions, stopCh <-chan struct{}) *MySQLClient {
+	m := &MySQLClient{}
+
+	conn, err := dbr.Open("mysql", fmt.Sprintf("%s:%s@tcp(%s)/devops?parseTime=1&multiStatements=1&charset=utf8mb4&collation=utf8mb4_unicode_ci", option.Username, option.Password, option.Host), nil)
```

Comment:
The specific SCHEMA used by the module is currently included in the link information. Do we need to move the mysql configuration to a different module?